app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java

```diff
@@ -76,15 +102,6 @@ public void askQuestionPopupIsDisplayed() {
         onView(ViewMatchers.withId(R.id.ask_question_popup)).check(matches(isDisplayed()));
     }
 
-    @Test
```

Comment:
There

Reply:
Yes, it is already tested in LoginTests, it was redundant.

Reply:
But they test two different buttons no ?